src/test/java/seedu/taassist/logic/commands/CommandTestUtil.java

```diff
@@ -62,10 +62,10 @@ public class CommandTestUtil {
     static {
         DESC_AMY = new EditStudentDescriptorBuilder().withName(VALID_NAME_AMY)
                 .withPhone(VALID_PHONE_AMY).withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY)
-                .withTags(VALID_TAG_FRIEND).build();
+                .withModuleClasses(VALID_TAG_FRIEND).build();
```

Comment:
The constants are not renamed yet. Similar to other files around this.

Reply:
a suggestion may be to refactor all the `TAG_FRIEND`/`TAG_HUSBAND`/`TAG_DESC_FRIEND`/`TAG_DESC_HUSBAND` to `CLASSES`

Reply:
Will do all these in another issue.